app/controllers/metrics_controller.rb

```diff
@@ -3,7 +3,13 @@ class MetricsController < ApplicationController
     # This method returns metrics ordered by wins and returns the top 10.
     def index
         @metrics = Metric.order(wins: :desc)
+        @user = User.where(user.id == metric.user_id)
         render json: @metrics
+        render json: @user
+    end
+
+    def show
+        @metric = Metric.where
```

Comment:
Looks like you forgot something here.   Also, please clear out any commented out code or console.logs.